docs/tutorials/create_expectations.rst

```diff
@@ -0,0 +1,358 @@
+
+
+
+::
+
+    context = ge.data_context.DataContext()
+
+To read more about DataContext, see: :ref:`data_context`
+
+
+
+2. List Data Assets
+-------------------
+
+A Data Asset is data you can describe with expectations.
+
+
+.. content-tabs::
+
+    .. tab-container:: tab0
+        :title: pandas
+
+        A Pandas datasource generates data assets from Pandas DataFrames or CSV files. In this example the pipeline processes NPI data that it reads from CSV files in ``npidata`` directory into Pandas DataFrames. This is the data you want to describe and specify with expectations. That directory and its files are a data asset, named "NPI data" (based on the directory name).
+
+    .. tab-container:: tab1
+        :title: pyspark
+
+        A Spark datasource generates data assets from Spark DataFrames or CSV files. In this example the pipeline processes NPI data that it reads from CSV files in ``npidata`` directory into Pandas DataFrames. This is the data you want to describe and specify with expectations. If the example read the data into Spark DataFrames, we would think of this data asset as "data from the npidata directory that we read into Spark DataFrames" and give it a name "NPI data".
+
+    .. tab-container:: tab2
+        :title: SQLAlchemy
+
+        A SQLAlchemy datasource generates data assets from tables, views and query results.
+
+        * If the data resided in a table (or view) in a database, it would be accessible as a data asset with the name of that table (or view).
+        * If the data did not reside in one table ``npidata`` and, instead, the example pipeline ran an SQL query that fetched the data (probably from multiple tables), the result set of that query would be accessible as a data asset. The name of this data asset would be up to us (e.g., "npidata" or "npidata_query").
+
+
+Use this convenience method to list all data assets and expectation suites in your project (using the `DataContext`).
+
+.. code-block:: python
+
+    great_expectations.jupyter_ux.list_available_data_asset_names(context)
+
+The output looks like this:
+
+.. image:: ../images/list_data_assets.png
+    :width: 600px
+
+``npidata`` is the short name of the data asset. Full names of data assets in a DataContext consist of three parts, for example: ``data__dir/default/npidata``. You don't need to know (yet) how the namespace is managed and the exact meaning of each part. The :ref:`data_context` article describes this in detail.
+
+
+3. Pick a data asset and set the expectation suite name
+-------------------------------------------------------
+
+The ``normalize_data_asset_name`` method converts the short name of a data asset to a full name:
+
+.. code-block:: python
+
+    data_asset_name = "npidata"
+    normalized_data_asset_name = context.normalize_data_asset_name(data_asset_name)
+    normalized_data_asset_name
+
+
+.. code-block:: python
+
+    expectation_suite_name = "warning"
+
```

Comment:
Slide this code block down to be below the description of name on lines 137-138

Reply:
Adding for post 0.8.4, since this requires synchronizing between the notebooks and the rst file.